services/pull/pull.go

```diff
@@ -64,7 +64,7 @@ func NewPullRequest(ctx context.Context, opts *NewPullRequestOptions) error {
 	}
 
 	// user should be a collaborator or a member of the organization for base repo
-	if !issue.Poster.IsAdmin {
+	if (!issue.Poster.IsAdmin) && pr.Flow == issues_model.PullRequestFlowGithub {
```

Comment:
```suggestion
	canCreate := issue.Poster.IsAdmin || pr.Flow == issues_model.PullRequestFlowAGit
	if !canCreate {
```

But should we also check something like "repo read permission" for "agit" to create?

Reply:
> But should we also check something like "repo read permission" for "agit" to create?

looks user already can't create pull request in my local test.

![image](https://github.com/user-attachments/assets/4ad0a1b0-393d-45c5-a80e-2677ee4e2fbd)